sites/flowcontrolnetwork/server/styles/index.scss

```diff
@@ -30,3 +30,6 @@ $theme-site-header-breakpoints: (
 @import "../../node_modules/@base-cms/marko-web-theme-default/skins/orion/skin";
 // @todo this should be remove once contact us is in core
 @import "../../node_modules/@base-cms-websites/package-common/scss/contact-us-form";
+
+/* stylelint-disable-next-line */
+.site-navbar__items--tertiary .site-navbar__link svg { fill: $black; }
```

Comment:
You can change this for all tertiary nav items by setting these vars to `$black`:

```scss
$theme-site-navbar-tertiary-link-active-color
$theme-site-navbar-tertiary-link-color
$theme-site-navbar-tertiary-link-hover-color
```